src/platform/qt/Window.cpp

```diff
@@ -1429,6 +1429,18 @@ void Window::setupMenu(QMenuBar* menubar) {
 	}
 	m_config->updateOption("fastForwardRatio");
 
+	addGameAction(tr("Fast forward speed increment"), "fastForwardUp", [this] {
+		float newRatio = m_config->getOption("fastForwardRatio", 1.0f).toFloat() + 1.0f;
+		m_config->setOption("fastForwardRatio", QVariant(newRatio));
```

Comment:
You also need to clamp this to make sure you're not adding 1 to -1, going from unbounded to 0x, which would break a lot. You can do by checking if the new ratio is at least 3.0f.

Reply:
Added the clamp and changed the wording.